### DISCUSSION TITLE
Battery module not loading

### DISCUSSION THREAD

**Discussion by soy-dev1** [polybar/polybar#2849] [Q&A] [answered]
My battery module does not seem to be loading, I have triple checked syntax to make sure it is all correct and I made sure it was reading from the correct config file in my home directory.

my config for the battery module looks like this: 
```
[module/battery]
type = internal/battery

full-at = 99
low-at = 10
battery = BAT
adapter = AC
poll-interval = 5
```
I have checked ls -l /sys/class/power_supply/, with the following output:
```
lrwxrwxrwx 1 root root 0 Oct 24 10:32 AC -> ../../devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:00/PNP0C09:00/ACPI0003:00/power_supply/AC
lrwxrwxrwx 1 root root 0 Oct 24 10:32 BAT0 -> ../../devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:00/PNP0C09:00/PNP0C0A:00/power_supply/BAT0
lrwxrwxrwx 1 root root 0 Oct 24 10:32 BAT1 -> ../../devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:00/PNP0C09:00/PNP0C0A:01/power_supply/BAT1
```
I am trying to get both batteries to display, but I am just starting with getting one to display. I have tried changing the `battery = BAT` line to `battery = BAT0` and `battery = BAT1`, but neither seem to work. 

at this point I am completely stuck. Here is the output when I run polybar manually through the terminal: 
```
notice: Parsing config file: /home/username/.config/polybar/config.ini
notice: Listening for IPC messages (PID: 5122)
notice: Loading module 'xworkspaces' of type 'internal/xworkspaces'
notice: Loading module 'xwindow' of type 'internal/xwindow'
notice: Loading module 'filesystem' of type 'internal/fs'
notice: Loading module 'pulseaudio' of type 'internal/pulseaudio'
notice: pulseaudio: using default sink alsa_output.pci-0000_00_1f.3.analog-surround-40
notice: Loading module 'xkeyboard' of type 'internal/xkeyboard'
notice: Loading module 'memory' of type 'internal/memory'
notice: Loading module 'cpu' of type 'internal/cpu'
notice: Loading module 'wlan' of type 'internal/network'
notice: module/wlan: Discovered wireless interface wlan0
notice: Loading module 'eth' of type 'internal/network'
notice: module/eth: Discovered wired interface enp0s31f6
notice: Loading module 'date' of type 'internal/date'
notice: Loaded 10 modules
notice: Loaded font "monospace" (name=Nimbus Mono PS, offset=2, file=/usr/share/fonts/gsfonts/NimbusMonoPS-Regular.otf)
```
if anyone has any ideas, help would be greatly appreciated. Thank you.

**Comment by patrick96** [comment, marked as answer]
Polybar never even tries load the module. You probably didn't add it to any of the module lists https://github.com/polybar/polybar/wiki/Configuration#module-settings

**Reply by soy-dev1** [reply]
I put my config into pastebin:
https://pastebin.com/XxfqU2pn

**Reply by patrick96** [reply]
Yeah, as I thought. You're not putting the name of the battery module into any of the three module lists (`modules-left`, etc.)

**Reply by soy-dev1** [reply]
Thanks! It's loading the module now.

**Reply by Netwatch-APT** [reply]
Thank You! This fixed it for me!!
